model/src/main/scala/org/broadinstitute/dsde/rawls/model/ExecutionModel.scala

```diff
@@ -435,7 +435,6 @@ trait ExecutionJsonSupport extends JsonSupport {
 
   implicit val SubmissionFormat = jsonFormat17(Submission)
 
-
```

Comment:
All of the changes in this file are `scalafmt`. I don't know why the formatting in this file passed in #1940, but is being flagged in this PR which I rebase-merged on top of #1940. But here it is, formatting is updated.